tasks/configure.yml

```diff
@@ -4,6 +4,21 @@
     name: bundler
   become_user: git
 
-- import_tasks: gitlab_permissions.yml
```

Comment:
Are you sure? There are many tasks in there. Otherwise, delete the unused file

Reply:
its not unused its used in install